address-sanitizer/tools/kasan_symbolize.py

```diff
@@ -151,6 +151,7 @@ def ProcessLine(self, line, lines_before, lines_after):
     for i, frame in enumerate(frames):
       inlined = (i + 1 != len(frames))
       func, fileline = frame[0], frame[1]
+      fileline = fileline.split(' ')[0] # strip '(discriminator N)'
```

Comment:
strictly saying file names can contain spaces
we need something more elaborated